src/main/java/org/profit/telegrambot/MaksymenBot.java

```diff
@@ -0,0 +1,169 @@
+            SendMessage sendMessage = new SendMessage();
+            sendMessage.enableMarkdown(true);
+            sendMessage.setText(textNewMsg);
+            sendMessage.setChatId(chatId);
+            ReplyKeyboardMarkup keyboardMarkup =
+                    buttonService.setButtons(buttonService.createButtons(asList(
+                            "Хліби", "Торти", "Тістечка", "Круасани"
+                    )));
+            sendMessage.setReplyMarkup(keyboardMarkup);
+            try {
+                execute(sendMessage);
+            } catch (TelegramApiException e) {
+                e.printStackTrace();
+            }
+        }
+
+        private void sendInlineKeyboard (String product, String chatId, String firstName, String secondName) {
+            SendMessage sendMessage = new SendMessage();
+            sendMessage.setText(product);
+            sendMessage.setChatId(chatId);
+            List<InlineKeyboardButton> firstRow = new LinkedList<>();
+            InlineKeyboardButton inlineKeyboardButtonFirst = new InlineKeyboardButton();
+            inlineKeyboardButtonFirst.setText(firstName);
+            inlineKeyboardButtonFirst.setCallbackData("first");
+            InlineKeyboardButton inlineKeyboardButtonSecond = new InlineKeyboardButton();
+            inlineKeyboardButtonSecond.setText(secondName);
+            inlineKeyboardButtonSecond.setCallbackData("second");
+                    /*
+                    List<InlineKeyboardButton> secondRow = new LinkedList<>();
+                    InlineKeyboardButton inlineKeyboardButtonThird = new InlineKeyboardButton();
+                    inlineKeyboardButtonSecond.setText(thirdName);
+                    inlineKeyboardButtonSecond.setCallbackData("third");
+                    InlineKeyboardButton inlineKeyboardButtonFourth = new InlineKeyboardButton();
+                    inlineKeyboardButtonSecond.setText(fourthName);
+                    inlineKeyboardButtonSecond.setCallbackData("fourth");
+                    List<InlineKeyboardButton> thirdRow = new LinkedList<>();
+                    InlineKeyboardButton inlineKeyboardButtonFifth = new InlineKeyboardButton();
+                    inlineKeyboardButtonSecond.setText(fifthName);
+                    inlineKeyboardButtonSecond.setCallbackData("fifth");
+                     */
+            firstRow.add(inlineKeyboardButtonFirst);
+            firstRow.add(inlineKeyboardButtonSecond);
+                    /*
+                    secondRow.add(inlineKeyboardButtonThird);
+                    secondRow.add(inlineKeyboardButtonFourth);
+                    thirdRow.add(inlineKeyboardButtonFifth);
+                     */
+            List<List<InlineKeyboardButton>> rowCollection = new LinkedList<>();
+            rowCollection.add(firstRow);
+                    /*
+                    rowCollection.add(secondRow);
+                    rowCollection.add(thirdRow);
+                     */
```

Comment:
The commented code could make your teammates a bit overwhelmed 🙂.

Could you please go through the changes and remove the commented code?